lib/assessment/rubric_tester.py

```diff
@@ -336,14 +355,16 @@ def main():
         response_type = params.get('response-type', 'tsv')
         prompt, standard_rubric = read_inputs(prompt_file, standard_rubric_file, params_lesson_prefix)
         student_files = get_student_files(options.max_num_students, dataset_lesson_prefix, student_ids=options.student_ids)
-        if os.path.exists(os.path.join(dataset_lesson_prefix, actual_labels_file_old)):
-            actual_labels = get_actual_labels(actual_labels_file_old, dataset_lesson_prefix)
-        else:
-            actual_labels = get_actual_labels(actual_labels_file, dataset_lesson_prefix)
+        if not options.new_labels:
+            if os.path.exists(os.path.join(dataset_lesson_prefix, actual_labels_file_old)):
+                actual_labels = get_actual_labels(actual_labels_file_old, dataset_lesson_prefix)
+            else:
```

Comment:
I think we just needed these lines during the transition from the old name to the new name. these 3 lines should be safe to remove now.